StRoot/StDAQMaker/StSCReader.h

```diff
@@ -57,6 +58,7 @@ class StSCReader
     sc_t *fSC;
     short flipBBCBkg;
     short useNoKillers;
+    short useEPD;
```

Comment:
There are a few good reasons to use the `bool` type for a variable intended to represent a true/false value in c++.